src/components/measurements/measurementsD3.js

```diff
@@ -305,7 +305,12 @@ export const drawMeansForColorBy = (ref, svgData, treeStrainColors) => {
     // 2 x subplotPadding for padding around the overall mean display
     const ySpacing = (layout.subplotHeight - 4 * layout.subplotPadding) / (numberOfColorByAttributes - 1);
     let yValue = layout.subplotPadding;
-    Object.entries(colorByGroups).forEach(([attribute, {color, values}]) => {
+    // Order the color groups by the legend value order so that we have a stable order for the means
```

Comment:
Did we ever consider doing this for the raw dots?

Reply:
Yup, we've discussed ordering the raw dots by color before (it's part of #1463). 
I think that will be a bigger change because of the jitter. Currently, the jitter is calculated during loading of the sidecar JSON, so will need to think through how to rework that...

Reply:
> I think that will be a bigger change because of the jitter. Currently, the jitter is calculated during loading of the sidecar JSON, so will need to think through how to rework that...

Thanks for the reference. I agree it's not for this PR, it just became apparent while reviewing it 😄